regression/contracts/invar_assigns_opt/test.desc

```diff
@@ -0,0 +1,20 @@
+CORE
+main.c
+--apply-loop-contracts
+^EXIT=0$
+^SIGNAL=0$
+^\[main.1\] .* Check loop invariant before entry: SUCCESS$
+^\[main.2\] .* Check that loop invariant is preserved: SUCCESS$
+^\[main.assertion.1\] .* assertion r1 == 0: SUCCESS$
+^\[main.assertion.2\] .* assertion s1 == 1: SUCCESS$
+^\[main.3\] .* Check loop invariant before entry: SUCCESS$
+^\[main.4\] .* Check that loop invariant is preserved: SUCCESS$
+^\[main.assertion.3\] .* assertion r2 == 0: SUCCESS$
+^\[main.assertion.4\] .* assertion s2 == 1: SUCCESS$
+^VERIFICATION SUCCESSFUL$
+--
+--
+This test checks that adding assigns clause is optional
```

Comment:
Could you rebase this branch and add a test case using an empty assigns clause (e.g., `__CPROVER_assigns()`)?